graylog2-web-interface/src/components/navigation/NavigationItem.tsx

```diff
@@ -71,7 +71,7 @@ const PluginNavDropdown = ({ menuItems, description, BadgeComponent, pathname }:
       {accessibleMenuItems.map((menuItem) => (
         <NavigationLink
           key={menuItem.description}
-          Badge={menuItem.BadgeComponent}
+          Badge={BadgeComponent}
```

Comment:
With this change, we will have the same error like we had before 
<img width="219" alt="Screenshot 2025-03-03 at 15 06 07" src="https://github.com/user-attachments/assets/18c9a2f3-7f93-4e2d-bc26-85537591ee99" />